app/src/main/java/ch/epfl/favo/favor/FavorUtil.java

```diff
@@ -23,14 +30,22 @@ public static FavorUtil getSingleInstance() {
   /**
    * Allows user to post a favor with a title, description and location.
    *
-   * @param title Title of favor.
-   * @param description String containing 300 char (max) description of text.
-   * @param location Address or coordinates at which the favor is requested. TODO: post favor in DB
-   *     linked to user
+   * @param f A favor object.
    */
-  public void postFavor(String title, String description, String location) {
-
-    throw new NotImplementedException();
+  public void postFavor(Favor f) {
+    if (testUiMode) {return;} //will skip if testing the UI
```

Comment:
Bad idea to have test code in production codebase, you should find another way. (e.g. a dependency injection in the collectionWrapper)